newsfragments/aip-79.significant.rst

```diff
@@ -0,0 +1,11 @@
+Remove Flask App Builder from core Airflow dependencies.
+
+As part of this change the following breaking changes have occurred:
+
+- The auth manager interface ``base_auth_manager`` have been updated with some breaking changes:
+
+  - The constructor no longer take ``appbuilder`` as parameter. The constructor takes no parameter
+
+  - A new abstract method ``deserialize_user`` needs to be implemented
+
+  - A new abstract method ``serialize_user`` needs to be implemented
```

Comment:
Those are not related to this PR but I forgot to mention them when adding them to the interface